notebooks/excerpt_overlap_review.ipynb

```diff
@@ -0,0 +1,262 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    print(f"Total Excerpts: {n_excerpts:,}\n")
</pre><p>These counts are great!</p><p> </p><p>If it's not too much trouble, it would be nice to see a handful of examples of each (maybe 10 or so same poem overlaps and 10 or so different poems?). Seeing how it looks when you've aggregated it would help me understand the structure better. (Although seeing the overlaps preview above, maybe getting the excerpt text back in to display is more trouble than it's worth at this point.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Princeton-CDH/ppa-nlp/pull/215/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>